### ISSUE TITLE
Scanning a CycloneDX sBOM results in No package sources found error

### ISSUE THREAD

**Issue by ccYHM86** [google/osv-scanner#355]
As of version 1.3.0, when scanning a CycloneDX generated sbom the osv-scanner tool shows the following error:
No package sources found, --help for usage information. When using version 1.2.0 the scan is successful.

Steps to Reproduce

1. Generate a CycloneDX sBOM (We use the following: https://www.npmjs.com/package/%40cyclonedx/cyclonedx-npm)
2. Use osv-scanner to scan the sbom (/go/bin/osv-scanner sbom.json)

**Expected Results**

osv-scanner should scan the sbom and report on vulnerabilities

**Actual Results**

```
root@cec9522aeabf:/build# /go/bin/osv-scanner --sbom auditjs.json
No package sources found, --help for usage information.
root@cec9522aeabf:/build# echo $?
128
```

For reference the sBOM I used is attached

[auditjs.json.zip](https://github.com/google/osv-scanner/files/11279215/auditjs.json.zip)

**Comment by another-rex** [comment]
This is likely caused by v1.3.0 now also identifying/filtering cyclonedx sboms by file names (i.e. https://cyclonedx.org/specification/overview/#recognized-file-patterns).

#339 has been merged recently and will be included in the next release (scheduled for next week) to skip the file name filtering if passed directly with the `--sbom` flag.

**Comment by another-rex** [comment]
Fixed by 1.3.2 release
